01-hello-world/helloWorld.js

```diff
@@ -4,7 +4,8 @@
 
 function sayHello(name)
 {
-    // Code goes here
+    var print = `Hello, ${name}!`;
```

Comment:
First, we shouldn't use `var` in modern code, use either `const` or `let`; in this case, it would be `const`.

Second, `print` is a terrible variable name and could be confused with a library or keyword.